src/modules/UI/components/ControlPanel/Component/Main.js

```diff
@@ -88,13 +88,16 @@ export default class Main extends React.Component<Props> {
   }
 }
 
-const goToScene = (scene: string) => {
+const goToScene = (scene: string, sceneProps?: any) => {
   const { currentScene, drawerClose } = Actions
-  if (currentScene === scene) {
-    drawerClose()
-  } else {
-    Actions.jump(scene)
+
+  if (currentScene !== scene) {
+    Actions.jump(scene, sceneProps)
+  } else if (sceneProps) {
+    Actions.refresh(sceneProps)
   }
+
+  drawerClose()
```

Comment:
No, I need it. Because when you try to navigate between two menu options `Sweep Private Key` and `Scan QR (Edge Login) `you get a bug, new button` Enter Key` will be visible on both of them. This is happen because `Sweep Private Key` and `Scan QR` use same `ScanScene` component and when you use `Actions.jump` nothing happens(even component rerender) that's why I use `Actions.refresh(sceneProps)`. Secondary, even if I rerender  `ScanScene` I get the same `data: 'sweepPrivateKey'` in props inside `Scan QR` route(if I previously  was on `Sweep Private Key` route).
In this case I add line `const popToSendScan = () => goToScene(Constants.SCAN, { isSweepPrivateKey: false })`

Reply:
👍🏼